GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -3144,6 +3164,94 @@ void AddSettingsItem()
                     };
                     commitTemplatesToolStripMenuItem.DropDownItems.Add(settingsItem);
                 }
+
+                void AddConventionalCommitsItems()
+                {
+                    _conventionalCommitItem = new(_conventionalCommit.Text, Images.GitCommandLog);
+
+                    foreach (string conventionKeyword in _headerCommitTypes)
+                    {
+                        ToolStripMenuItem commitTypeMenuItem = new(conventionKeyword, null, (_, _) =>
+                        {
+                            (string message, int selectionStart) = PrefixOrReplaceKeyword(conventionKeyword);
+                            Message.Text = message;
+                            Message.SelectionStart = selectionStart;
+                            Message.Focus();
+                        });
+
+                        if (commitTypeMenuItem.Text == Feat)
+                        {
+                            commitTypeMenuItem.ShortcutKeyDisplayString = (Keys.Control | Keys.T).ToShortcutKeyDisplayString();
+                        }
+
+                        if (commitTypeMenuItem.Text is Feat or Fix)
+                        {
+                            commitTypeMenuItem.Font = new Font(commitTypeMenuItem.Font, FontStyle.Bold);
```

Comment:
Why bold for just these two?

Reply:
I think that are the main 2 ones used.  So the goal is to catch the eye because there are in the middle of the alphabetically sorted keyword list.

Reply:
Bold items in context menus generally represent defaults. I would remove the bold here. The _Ctrl + T_ text next to the _Feat_ item draws the eye, I think.

Reply:
done (even if I found it clearer with...)